pkgs/args/lib/command_runner.dart

```diff
@@ -33,7 +33,15 @@ class CommandRunner<T> {
   ///
   /// Defaults to `"$executableName <command> arguments`". Subclasses can
   /// override this for a more specific template.
-  String get invocation => '$executableName <command> [arguments]';
+  String get invocation {
+    var command = '<command>';
+
+    if (commands.containsKey(Command.defaultSubcommand)) {
+      command = '[$command]';
+    }
```

Comment:
Is this branch covered in tests? I didn't find any strings that looked like they would come from `[$command]`